packages/xo-web/src/xo-app/backup/restore/metadata.js

```diff
@@ -32,21 +32,27 @@ const restore = entry =>
     }),
     body: <RestoreMetadataBackupModalBody backups={entry.backups} type={entry.type} />,
     icon: 'restore',
-  }).then(backup => {
-    if (backup === undefined) {
+  }).then(data => {
+    if (data === undefined) {
       error(_('backupRestoreErrorTitle'), _('chooseBackup'))
       return
     }
-    return restoreMetadataBackup(backup)
+    return restoreMetadataBackup({ backup: data.backup.id, pool: data.pool.id })
```

Comment:
You can use `resolveId`.
```suggestion
    return restoreMetadataBackup({ backup: resolveId(data.backup), pool: resolveId(data.pool) })
```